.github/workflows/pytest.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.9]
```

Comment:
Is this because new gym requires 3.9?

Reply:
No, this was me trying to figure out why all the unit tests were failing in 3.7 and 3.8. I will restore that now that I've figured it out.